app/ide-desktop/lib/dashboard/src/layouts/Editor.tsx

```diff
@@ -78,12 +78,21 @@ export default function Editor(props: EditorProps) {
       void (async () => {
         const jsonAddress = project.jsonAddress
         const binaryAddress = project.binaryAddress
-        const ydocAddress = ydocUrl ?? ''
         if (jsonAddress == null) {
           toastAndLog('noJSONEndpointError')
         } else if (binaryAddress == null) {
           toastAndLog('noBinaryEndpointError')
         } else {
+          let ydocAddress: string
+          if (ydocUrl == null) {
+            ydocAddress = 'undefined'
```

Comment:
Again unexpected stringified `undefined` that I'd like to get rid of.

Reply:
This special value can be confusing, I agree. I refactored the `ydocUrl` logic